gui.py

```diff
@@ -71,6 +77,10 @@ def __init__(self):
         GPSConnectedEvent.register_handler(self)
         GPSDisconnectedEvent.register_handler(self)
 
+    def present_main_app(self):
+        self.splash.destroy()
+        self.app.visible = True
```

Comment:
we're never going to display the splash screen again, so we can destroy it. This way its memory is reclaimed.